src/test/groovy/org/prebid/server/functional/testcontainers/container/PrebidServerContainer.groovy

```diff
@@ -23,6 +32,7 @@ class PrebidServerContainer extends GenericContainer<PrebidServerContainer> {
     PrebidServerContainer(String dockerImage, Map<String, String> customConfig) {
         super(dockerImage)
         withExposedPorts(PORT, DEBUG_PORT, ADMIN_PORT)
+        withExposedPortsDebug()
```

Comment:
Words order. You are not exposing ports to debug, you want to expose debugging ports. Rename pls